eslzArm/eslz-portal.json

```diff
@@ -1293,6 +1293,292 @@
                                 ]
                             }
                         },
+                        {
+                            "name": "privateDnsZones",
+                            "type": "Microsoft.Common.DropDown",
+                            "label": "Select Private DNS Zones to create",
+                            "defaultValue": "",
+                            "multiselect": true,
+                            "selectAll": true,
+                            "filter": true,
+                            "multiLine": true,
+                            "visible": "[and(or(equals(steps('connectivity').enableHub, 'vhub'), equals(steps('connectivity').enableHub, 'nva')), not(equals(steps('connectivity').enablePrivateDnsZones,'No')))]",
+                            "toolTip": "Select each Private DNS Zone to create which will offer DNS integration with Azure PaaS services where you enable private endpoints.",
+                            "constraints": {
+                                "allowedValues": [
```

Comment:
How are we handling the zones for Azure Backup & Site Recovery? These today get added and the correct geo code added to the DNS zone name, instead of the region short name.

This happens in the variable called `privateDnsZonesMerge` which still exists in this PR, which I'd expect it to 👍 But we need to work out how we can add some logic to this drop down that allows customers to enable/disable the backup and site recovery zones being created, just as the other zones in this dropdown.

Happy to chat further offline, but we need to investigate here

Reply:
@jtracey93 I'd love to have an offline chat to further discuss that one which might also help me with the following bug as well as it's including Azure Backup & Site Recovery: [Bug Report: Deploy Private DNS Zones - Built-in policies missing from initiative](https://github.com/Azure/Enterprise-Scale/issues/1485)

Reply:
Setup some time for us early next year 👍